remove `.` from moving rollup (#8474)
DataDog/documentation: content/en/dashboards/functions/rollup.md

|------------------|------------------------------------------------|------------------|
| `moving_rollup` | Rollup to combine the points in the last X seconds. | `moving_rollup(<METRIC_NAME>, <TIME> , <METHOD>)` |

> Applying the `moving_rollup()` function to a query allows you to combine points from the most recent specified time range, i.e. the last X seconds. Like with `.rollup()`, `<METHOD>` can be `sum`/`min`/`max`/`count`/`avg` and defines how data points are aggregated within the given time interval.

Reviewer: ```suggestion
Applying the `moving_rollup()` function to a query allows you to combine points from the most recent specified time range—that is, the last X seconds. Like with `.rollup()`, `<METHOD>` can be `sum`/`min`/`max`/`count`/`avg` and defines how data points are aggregated within the given time interval.
```
